lighthouse-core/config/config.js

```diff
@@ -568,6 +569,11 @@ class Config {
   get categories() {
     return this._categories;
   }
+
+  /** @type {Object<string, {title: string, description: string}>} */
```

Comment:
if we're bein' real it's `*` for most of these defs since the config can stick whatever it wants here ;) but yeah could be undefined I'll add